Lib/test/test_shutil.py

```diff
@@ -34,7 +34,7 @@
 
 TESTFN2 = TESTFN + "2"
 MACOS = sys.platform.startswith("darwin")
-AIX = sys.platform[:3] == 'aix'
+AIX = sys.platform in ('aix',)
```

Comment:
```suggestion
AIX = sys.platform == 'aix'
```

This should either be left as-is or changed to equality unless the list is going to contain more than one item.

Reply:
I think we should revert this to how it was because `sys.platform` might return version prefix after aix. So checking the substring is better.

```python
>>> import sys
>>> sys.platform
'aix7'
```

Reply:
That was changed in Python 3.8 to return only "aix"

Reply:
https://docs.python.org/3/library/sys.html#sys.platform

> Changed in version 3.8: On AIX, sys.platform doesn’t contain the major version anymore. It is always 'aix', instead of 'aix5' or 'aix7'. Since older Python versions include the version number, it is recommended to always use the startswith idiom presented above.

Note the recommendation wouldn't need to apply to the tests shipped with Python, since those are tied to the version of Python being built, which is known